c10/core/DispatchKey.cpp

```diff
@@ -239,7 +239,35 @@ const char* toString(DispatchKey t) {
       return "AutogradFunctionality";
 
     default:
-      return "UNKNOWN_TENSOR_TYPE_ID";
+      auto bc = toBackendComponent(t);
+      auto fk = toFunctionalityKey(t);
+#define FORALL_BC(fk) \
+  switch (bc) { \
+    case BackendComponent::CPUBit: return fk "CPU"; \
```

Comment:
can we use https://github.com/pytorch/pytorch/blob/adf806060054c31904f0b180da0daa058ba24793/c10/core/DispatchKey.cpp#L8-L11 to avoid duplication?

Reply:
I guess we can refactor toString to have a version that doesn't concatenate "Bit" on and then use that. Really don't want Bit in the print here.